src/components/templates/layout.js

```diff
@@ -14,9 +14,15 @@ const Layout = () => {
     }
   `)
 
-  return (
-    <NavBar siteTitle={data.site.siteMetadata.title} />
-  )
+  return <NavBar isHome={isHome} siteTitle={data.site.siteMetadata.title} />
+}
+  
+Layout.propTypes = {
+  isHome: PropTypes.Bool,
```

Comment:
```suggestion
  isHome: PropTypes.bool,
```
Also looks like we forgot to import : 
```
import PropTypes from "prop-types"
```